nUpdate/UI/Dialogs/MissingRequirementDialog.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Author: Dominic Beger (Trade/ProgTrade)
+
+using System;
+using System.Drawing;
+using System.IO;
+using System.Reflection;
+using System.Windows.Forms;
+using nUpdate.Core;
+using nUpdate.Core.Localization;
+
+namespace nUpdate.UI.Dialogs
+{
+    public partial class MissingRequirementDialog : BaseDialog
+    {
+        private LocalizationProperties _lp;
+        private readonly Icon _appIcon = Icon.ExtractAssociatedIcon(Application.ExecutablePath);
+
+        public MissingRequirementDialog()
+        {
+            InitializeComponent();
+        }
+
+        /// <summary>
+        ///     Sets the name of the _lpuage file in the resources to use, if no own file is used.
+        /// </summary>
+        public string LanguageName { get; set; }
+
+        /// <summary>
+        ///     Sets the path of the file which contains the specific _lpuage content a user added on its own.
+        /// </summary>
+        public string LanguageFilePath { get; set; }
+
+        private void MissingRequirementDialog_Load(object sender, EventArgs e)
+        {
+            if (!String.IsNullOrEmpty(LanguageFilePath))
+            {
+                try
+                {
+                    _lp = Serializer.Deserialize<LocalizationProperties>(File.ReadAllText(LanguageFilePath));
+                }
+                catch (Exception)
+                {
+                    _lp = new LocalizationProperties();
+                }
+            }
+            else if (String.IsNullOrEmpty(LanguageFilePath) && LanguageName != "en")
+            {
+                string resourceName = String.Format("nUpdate.Core.Localization.{0}.json", LanguageName);
+                using (Stream stream = Assembly.GetExecutingAssembly().GetManifestResourceStream(resourceName))
+                {
+                    _lp = Serializer.Deserialize<LocalizationProperties>(stream);
+                }
+            }
+            else if (String.IsNullOrEmpty(LanguageFilePath) && LanguageName == "en")
+            {
+                _lp = new LocalizationProperties();
+            }
+
+            closeButton.Text = _lp.CloseButtonText;
+            headerLabel.Text = _lp.MissingRequirementDialogHeader;
+            infoLabel.Text = _lp.MissingRequirementDialogInfoText;
+
```

Comment:
This does not help too much. It would be better if you collected the missing requirements for each new version and listed it here.